### PR TITLE
EVG-1936 fix URL for CLI API V2 Requests

### PR DESCRIPTION
**Pull request by richardsamuels**

### CODE REVIEW

File: cli/http.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func getAPIClients(ctx context.Context, o *Options) (*APIClient, *APIClient, *mo
 
 	ac := &APIClient{
 		APIRoot:   settings.APIServerHost,
-		APIRootV2: apiURL.Scheme + "://" + apiURL.Host + "/rest/v2",
+		APIRootV2: settings.APIServerHost + "/rest/v2",
```
Comment (tychoish) [inline cli/http.go]: why only make this change here and not also on line 81? 

what is the impact of this change, it looks ~isomorphic?

(tbh I think the legacy CLI tool's api client is weird and should be refactored/removed at some point, but I think that some point may be in the future.)

Comment (richardsamuels) [reply cli/http.go]: Say the evergreen config has "api_server_host: https://evergreen.mongodb.com/api" in it.

For `ac`, the APIRoot and APIRootV2 will be "https://evergreen.mongodb.com/api" and "https://evergreen.mongodb.com/api/rest/v2" respectively.

For `rc`, they will be "https://evergreen.mongodb.com/" and "https://evergreen.mongodb.com/rest/v2" respectively.

So i'm actually fixing ac's APIRootV2 to be consistent with APIRoot.

Comment (bsamek) [reply cli/http.go]: https://evergreen.mongodb.com/api/rest/v2/hosts and https://evergreen.mongodb.com/rest/v2/hosts are both legitimate URLs that point to the same route, the first on :8080 and the second on :9090. I'm not sure that's the case for *all* API and UI routes though.

@john-m-liu, You looked at this this morning. Does this change make sense to you?

Comment (john-m-liu) [reply cli/http.go]: It does make sense to me. I think the bug with the previous implementation is that our API root when hitting the API port is not the same as the host for the url. This is made more confusing by us trying to combine the API/UI services, but currently we register the same routes at a different root in the UI server (see line 248 of ui.go). But for now I think this is correct

**Pull request merged**